src/baseTrie.js

```diff
@@ -50,7 +53,8 @@ module.exports = class Trie {
 
     this.root = root
 
-    this.putRaw = this._putRaw
+    this.getRaw = this.db.get.bind(this.db)
+    this.delRaw = this.db.del.bind(this.db)
```

Comment:
Update: after some more digging into the code and having made this `putRaw` - scratch vs permanent DB write - comment: this changes a bit the notion of this, updated question: is having these `raw` methods being used through `trie` and transparently working rather on scratch or permanent DB useful? Or should we rather go one step further and remove this here and eventually completely disallow the old raw `level` passing constructor here and tell people to switch to instantiate with `DB`?

Reply:
I'm not sure to be honest, this is my question too. Do you know how or if these `raw` methods are being used in the wild? Maybe that'd help us in making the decision. I'm more inclined towards removing the methods (but maybe because I'm not aware of its use-cases)

Reply:
Since the use cases respectively the functionality provided (writing to the permanent DB) stay the same when invoked through `DB` - I think we can remove these here and tell people to use through the DB class.

This would also play well with the suggestion above to force the instantiation with `DB`.